app/models/mixins/virtual_total_mixin.rb

```diff
@@ -16,7 +16,7 @@ module ClassMethods
     #
     def virtual_total(name, relation)
       define_method(name) do
-        send(relation).count
+        send(relation).try(:size)
```

Comment:
`count` works for arrays.

can we touch base tomorrow to resolve this.  we shouldn't have to chanhe this in the ui. I wad pretty sure it returned nil before. if not, then I need to fix in the model.